lib/moon_web/components/facing/card_article.ex

```diff
@@ -39,9 +39,9 @@ defmodule MoonWeb.Components.Facing.CardArticle do
 
   defp get_class(variant) do
     case variant do
-      "tall" -> "pr-9 w-[288px] h-[448px]"
-      "wide" -> "pr-12 w-[448px] h-[288px]"
-      _ -> "pr-12 w-[384px] h-[288px]"
+      "tall" -> "md:w-[18rem] md:h-[28rem] md:pe-9"
```

Comment:
https://tailwindcss.com/docs/width
w-[18rem] = w-72
24rem = 96
the same for height